### DISCUSSION TITLE
--include-threads for channels

### DISCUSSION THREAD

**Discussion by Viridescence** [Tyrrrz/DiscordChatExporter#1164] [Q&A] [answered]
Hello! I've noticed there seems to be a very handy --include-threads option for exportall, exportguild, and the channel lister... Is any equivalent implemented for the single-channel "export" command or would one need to script that themselves? The CLI seems to think not, and the docs are a little ambiguous on which functions actually accept the argument.

Thanks a ton!

**Comment by Tyrrrz** [comment, marked as answer]
That is currently not supported

Edit: now it is

**Reply by mjuanes** [reply]
Awesome tool, this capability would be really helpful. I'm downloading a huge amout of unnecesary content because I can't include threads for channels.

**Comment by 96-LB** [comment]
Update: this is now possible as of version 2.46
